pkg/cmd/admin/policy/reconcile_sccs.go

```diff
@@ -235,6 +236,10 @@ func (o *ReconcileSCCOptions) computeUpdatedSCC(expected kapi.SecurityContextCon
 		if actual.Priority != nil {
 			expected.Priority = actual.Priority
 		}
+
+		// preserve labels and annotations
+		util.AddObjectLabels(&expected, actual.Labels)
```

Comment:
I don't want modifications to expected to escape the function. This modifies the map in the `expected` obj passed in